esmvalcore/cmor/_fixes/native6/era5.py

```diff
@@ -0,0 +1,242 @@
+"""Fixes for ERA5."""
+import numpy as np
+import iris
+
+from ..fix import Fix
+from ..shared import add_scalar_height_coord
+
+
+class FixEra5(Fix):
+    """Fixes for ERA5 variables."""
+
+    @staticmethod
+    def _frequency(cube):
+
+        if not cube.coords(axis='T'):
+            return 'fx'
+        time = cube.coord(axis='T')
+        if len(time.points) == 1:
+            return 'fx'
+
+        interval = time.points[1] - time.points[0]
+        unit = 'hours' if 'hour' in time.units.name else 'days'
```

Comment:
Why the complicated logic trying to distinguish hours since from days since? You could simply convert the coordinate units to `'days since 1850-1-1 00:00:00.0'` if that hasn't been done yet by the time you reach this code